tests/test_parsing.py

```diff
@@ -67,7 +68,10 @@ def test_absolute_quality(self):
 
 class ScanningTest(TestCase):
     def test_scanning(self):
-        self.assertRaises(InterfaceError, Cell.all, 'fake-interface')
+        #self.assertRaises(InterfaceError, Cell.all, 'fake-interface')#test fails
+        with self.assertRaises(InterfaceError):
```

Comment:
can you remove the comments, and keep both tests: without sudo first, and with sudo after, both in their own `with` block (and maybe keeping the original syntax, i.e. without the `with`)